args/core/engine/module.hpp

```diff
@@ -27,6 +27,8 @@ namespace args::core
 		 * @ref ARGS_PURE
 		 */
 		virtual void init() ARGS_PURE;
+
+		virtual ~Module() ARGS_IMPURE;
```

Comment:
this should really be marked = default;

Reply:
resolution can be found in commit #59333b4